infra/notifications/migrations/20190502223858-create-notification-log.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const TableName = 'notification_log'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface
+      .createTable(TableName, {
+        id: {
+          type: Sequelize.INTEGER,
+          allowNull: false,
+          autoIncrement: true
+        },
+        send_date: {
```

Comment:
Do we need this send_at column or could we use created_at which typically stores the time the row was inserted in the DB ?

Reply:
Ah yeah, that would work! (Designed the table structure before realizing the Sequelize always adds those date columns)